### ISSUE TITLE
add get temperature.

### ISSUE THREAD

**Issue by rembozebest** [bensku/Skript#896]
doc: https://hub.spigotmc.org/javadocs/spigot/org/bukkit/block/Block.html#getTemperature--

example:
```
on join:
    message "temperature: %temperature%" # or
    message "temperature: %temperature at player%" # or
    message "temperature: %temperature of block above player%"
```

@bensku 
thanks to all.

**Comment by FUZIK** [comment]
wiki: https://minecraft.gamepedia.com/Biome#Temperature

**Comment by JRoy** [comment]
@Snow-Pyon - This can be closed as it's addressed in #1162

**Comment by TheBentoBox** [comment]
@JRoy Issues generally aren't closed until a release containing the feature/bug fix is officially published.
